apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2Test.java

```diff
@@ -128,6 +128,7 @@ public void mapEobCommonGroupInpOutHHAHospiceSNFWhenFiNumberExistsExpectExtensio
     TransformerUtilsV2.mapEobCommonGroupInpOutHHAHospiceSNF(
         eob,
         Optional.empty(),
+        Optional.empty(),
```

Comment:
We should have a test testing when this is not empty that the correct value is mapped.

Reply:
Done